gnocchi/carbonara.py

```diff
@@ -196,8 +196,8 @@ def quantile(self, q):
         ordered = numpy.lexsort((self._ts['values'], self.indexes))
         min_pos = numpy.cumsum(self.counts) - self.counts
         real_pos = min_pos + (self.counts - 1) * (q / 100)
-        floor_pos = numpy.floor(real_pos).astype(numpy.int, copy=False)
-        ceil_pos = numpy.ceil(real_pos).astype(numpy.int, copy=False)
+        floor_pos = numpy.floor(real_pos).astype(numpy.integer, copy=False)
```

Comment:
When was numpy.integer introduced? I.e do we need to have a new lowest supported version for numpy?

Reply:
It seems to be first introduced in 1.19.0. New lowest, I am not sure what you mean. Do you mean in `setup.cfg`

Reply:
I was wrong, it is already in 1.9.0. Therefore, I removed my change in lower contraints.